.travis.yml

```diff
@@ -6,18 +6,19 @@ node_js:
 sudo: false
 
 cache:
+  yarn: true
   directories:
-    - $HOME/.npm
     - $HOME/.cache # includes bowers cache
 
 before_install:
-  - npm config set spin false
-  - npm install -g bower phantomjs-prebuilt
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+  - yarn global add bower phantomjs-prebuilt
   - bower --version
   - phantomjs --version
 
 install:
-  - npm install
+  - yarn install
```

Comment:
we prefer `yarn install --no-lockfile` (see https://github.com/ember-cli/ember-cli/blob/master/blueprints/addon/files/.travis.yml#L35) for addons, to make sure you get notified when a transitive dependency update breaks your addon. otherwise you have to wait for your users to notify you.